import "define_pro.proto";
import "define_attrib.proto";

message MsgVLP
{
  optional string       username     = 1;
  optional string       password     = 2;
  optional fixed32      version      = 3;
  optional TAuthType    auth_type    = 4;
}

message MsgLogin
{
  optional string       token        = 1;
  optional fixed32      version      = 2;
}

message MsgCreatePlayer
{
  optional string       name         = 1;
}

message MsgLoadPlayer
{
  optional uint32       player_idx   = 1;
}

message MsgDBID
{
	optional fixed64    dbid         = 1;
}

message MsgID32
{
	optional uint32    id         = 1;
}



